src/interfaces/token.spec.ts

```diff
@@ -789,28 +790,61 @@ describe("TokenInterface", () => {
       );
       expect(approveMock).toHaveBeenCalledTimes(1);
       expect(approveMock).toHaveBeenCalledWith(spenderAddress, amount, {});
-      expect(sendTransactionMock).toHaveBeenCalledTimes(1);
-      expect(sendTransactionMock).toHaveBeenCalledWith("approved");
```

Comment:
Instead of deleting these expectations we could make sure it's not being called

```typescript
expect(sendTransactionMock).not.toHaveBeenCalled();
```

Reply:
since `sendTransaction` is not part of `populateApprove`, decided not to include it. Seems like github is acting weird with this one since its almost a copy of `approve`, which does includes it.

Reply:
Ah ok, I think that's fair, but wouldn't hurt to have it there as we kind of want this to be a dry run type of thing and not to actually send the transaction. I'm fine either way. Will resolve for now.